builtin/mainmenu/tab_texturepacks.lua

```diff
@@ -25,7 +26,6 @@ local function filter_texture_pack_list(list)
 	end
 
 	table.sort(retval)
-	table.insert(retval, 1, fgettext("None"))
```

Comment:
Wait a moment... you aren't supposed to make that.

`None` should remain at the top, not sorted alphabetically.